src/components/auth/RegisterChildren.vue

```diff
@@ -430,8 +431,10 @@ async function handleCheckCaptcha() {
   });
 }
 
-async function getConsent() {
-  showConsent.value = true;
+async function getConsent(outerIndex) {
+  const consentDoc = await authStore.getLegalDoc('consent-behavioral-eye-tracking');
+  consentText.value = consentDoc.text;
+  showConsent.value[outerIndex] = true;
   handleCheckCaptcha();
 }
```

Comment:
```suggestion
async function getConsent(outerIndex) {
try {
  const consentDoc = await authStore.getLegalDoc('consent-behavioral-eye-tracking');
  consentText.value = consentDoc.text;
  showConsent.value[outerIndex] = true;
  handleCheckCaptcha();
}
} catch(error) {
    console.error('Failed to fetch consent form', error);
    throw new Error('Could not fetch consent form');
}
```
This might be a good place to attach some error handling. If the students cannot assent/consent, then perhaps they should not be able to continue with the assessment since their data would be unusable to the researchers? Not sure whether we'd want to `throw` or `console.error`, or return something else in that case.

Just an idea.